src/EnergyPlus/UnitarySystem.cc

```diff
@@ -11776,6 +11802,45 @@ namespace UnitarySystems {
         //  UnitarySystem(UnitarySysNum)%SuppHeatPartLoadFrac = PartLoadRatio
     }
 
+    void UnitarySys::setEMSSuppCoilStagePLR(EnergyPlusData &state)
+    {
+        bool useMaxedSpeed = false;
+        int SpeedNumEMS = ceil(this->m_EMSOverrideSuppCoilSpeedNumValue);
+        if (SpeedNumEMS > this->m_NumOfSpeedSuppHeating) {
+            SpeedNumEMS = this->m_NumOfSpeedSuppHeating;
+            useMaxedSpeed = true;
+        }
```

Comment:
>= ?

Reply:
The difference of > and >= is just whether to throw a warning when the EMS speed value is equal to coil speed number (useMaxedSpeed is used to throw warnings). My understanding is that we should allow this situation without throwing warnings.